rct229/rules/section22/section22rule1.py

```diff
@@ -0,0 +1,107 @@
+
+
+class Section22Rule1(RuleDefinitionListIndexedBase):
+    """Rule 1 of ASHRAE 90.1-2019 Appendix G Section 22 (Hot water loop)"""
+
+    def __init__(self):
+        super(Section22Rule1, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section22Rule1.ChillerFluidLoopRule(),
+            index_rmr="baseline",
+            id="22-1",
+            description="Baseline chilled water design supply temperature shall be modeled at 44F.",
+            rmr_context="ruleset_model_instances/0",
+            list_path="fluid_loops[*]",
+        )
+
+    def create_data(self, context, data):
+        rmi_b = context.baseline
+        chillers = find_all("$.chillers[*]", rmi_b)
+        loop_chiller_dict = {}
+        for chiller_b in chillers:
+            loop_id = getattr_(chiller_b, "chillers", "cooling_loop")
+            if not loop_id in loop_chiller_dict.keys():
+                loop_chiller_dict[loop_id] = []
+            loop_chiller_dict[loop_id].append(chiller_b)
+        return {"loop_chiller_dict": loop_chiller_dict}
+
+    def is_applicable(self, context, data=None):
+        rmi_b = context.baseline
+        # FIXME: replace with baseline_system_types = get_baseline_system_types(rmi_b) when get_baseline_system_types
+        #  is ready.
+        baseline_system_types = {
+            "SYS-7": ["hvac_sys_7"],
+            "SYS-11-a": ["hvac_sys_11_a"],
+        }
+        # if any system type found in the APPLICABLE_SYS_TYPES then return applicable.
+        return any(
+            [key in APPLICABLE_SYS_TYPES for key in baseline_system_types.keys()]
+        )
+
+    def list_filter(self, context_item, data):
+        fluid_loop_b = context_item.baseline
+        loop_chiller_dict = data["loop_chiller_dict"]
+        return fluid_loop_b["id"] in loop_chiller_dict.keys()
+
+    class ChillerFluidLoopRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section22Rule1.ChillerFluidLoopRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+                required_fields={
+                    "$": ["cooling_or_condensing_design_and_control"],
+                    "cooling_or_condensing_design_and_control": [
+                        "design_supply_temperature"
+                    ],
+                },
+            )
+
+        def get_calc_vals(self, context, data=None):
+            fluid_loop_b = context.baseline
+            design_supply_temperature = fluid_loop_b[
+                "cooling_or_condensing_design_and_control"
+            ]["design_supply_temperature"]
+
+            return {"design_supply_temperature": design_supply_temperature}
+
+        def rule_check(self, context, calc_vals=None, data=None):
+            design_supply_temperature = calc_vals["design_supply_temperature"]
+            return std_equal(
+                design_supply_temperature.to(ureg.kelvin),
```

Comment:
Do we really need to convert both to Kelvin? I'm sure you've thought this through; I'm curious about the reasoning.